[ERROR   ] 15:43:07 rpcserver:303 Exception calling RPC request: '39acced0efd12db7673c815c877194f29a881838'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: '39acced0efd12db7673c815c877194f29a881838'
[ERROR   ] 15:43:15 rpcserver:303 Exception calling RPC request: '5994472f8ec52a30cfb843ff79189f5565278a55'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: '5994472f8ec52a30cfb843ff79189f5565278a55'
[ERROR   ] 15:43:20 rpcserver:303 Exception calling RPC request: 'a82201c7c3f8c5dce763efd60081f76cd5638179'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: 'a82201c7c3f8c5dce763efd60081f76cd5638179'
[ERROR   ] 15:43:45 rpcserver:303 Exception calling RPC request: '9257b549add6a79dc993aca52b1698584656b068'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: '9257b549add6a79dc993aca52b1698584656b068'
[ERROR   ] 15:43:58 rpcserver:303 Exception calling RPC request: '54d1733fe2126e2fc507a7025a276c1c82a513cb'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: '54d1733fe2126e2fc507a7025a276c1c82a513cb'
[ERROR   ] 15:44:07 rpcserver:303 Exception calling RPC request: '752edc5ab8786bf1f71ef2a17bfdc21a38ad72f7'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: '752edc5ab8786bf1f71ef2a17bfdc21a38ad72f7'
[ERROR   ] 15:44:11 rpcserver:303 Exception calling RPC request: '8f0439cdd5791730d85b44fd5a6463ee5d385033'
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/deluge/core/rpcserver.py", line 298, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 415, in get_torrents_status
    status_dict[torrent_id] = self.get_torrent_status(torrent_id, keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/core.py", line 397, in get_torrent_status
    status = self.torrentmanager[torrent_id].get_status(keys, diff)
  File "/usr/lib/pymodules/python2.6/deluge/core/torrentmanager.py", line 273, in __getitem__
    return self.torrents[torrent_id]
KeyError: '8f0439cdd5791730d85b44fd5a6463ee5d385033'
